packages/components/src/navigation/styles/navigation-styles.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * External dependencies
+ */
+import styled from '@emotion/styled';
+
+const horizontalPadding = '30px';
+
+export const Root = styled.div`
+	width: 272px;
+	padding: 35px 0px;
+	> *:not( .components-navigation__menu ) {
```

Comment:
I feel like this is a common situation, especially with cards and child items need to span the entire width. Has there been a precedent set in Gutenberg for this?

Reply:
Looking over the components it seems its about half that need to span the entire width and half that don't.  My thinking here was that it's more reliable for a consumer to set `0` padding/margin than it is to match existing default padding and margins (especially if these change over time).

>Has there been a precedent set in Gutenberg for this?

I think in general this component took too much liberty in styling, so I've scaled back on all styling and this has been removed.